src/platform/android/java/chip/platform/NsdManagerServiceBrowser.java

```diff
@@ -81,23 +89,30 @@ public void startDiscover(
       final long contextHandle,
       final ChipMdnsCallback chipMdnsCallback) {
     if (callbackMap.containsKey(callbackHandle)) {
-      Log.d(TAG, "Invalid callbackHandle");
+      Log.w(TAG, "Starting service discovering failed. Invalid callback ID: " + callbackHandle);
```

Comment:
Just call it "callbackHandle" instead of "callback ID" in the string because ID could mean something else and is confusing to see in this context.